packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts

```diff
@@ -145,10 +145,34 @@ const getBorderRadiusValue = (
     : makeBorderSize(getIn(theme, `border.radius.${responsiveBorderRadiusValue}`));
 };
 
+const getBorderWidthValue = (
```

Comment:
do we ever do responsive width for borders? 🤔

Reply:
Unlikely but consistency of API here is probably more important